gradebookng/tool/src/java/org/sakaiproject/gradebookng/GradebookNgApplication.properties

```diff
@@ -42,7 +42,7 @@ label.coursegrade.studentnotreleased=Not Yet Released
 
 label.gradeitem.externalapplabel=Gradebook Item coming from {0}
 label.gradeitem.released=Gradebook Item has been released and is visible to students.
-label.gradeitem.notreleased=Gradebook item is not visible to students.
+label.gradeitem.notreleased=Released Gradebook items are not visible to students.
```

Comment:
Is "Released" accurate? 

How about: "Gradebook items are not visible to students."